src/build/package-builder.js

```diff
@@ -26,12 +26,18 @@ export default class PackageInstallScripts {
   needsPermission: boolean;
   resolver: PackageResolver;
   reporter: Reporter;
-  installed: number;
+  installedCount: number;
   config: Config;
   force: boolean;
 
+  getPJCConfig(pkg: Manifest): ?Object {
+    const obj: mixed = pkg.pjc || pkg.esy;
```

Comment:
We have a bunch of packages already converted over, and we're just about ready to flip them over to the master branches, so that all the master branches will be ready for building on PJC spec. Currently, their `package.json` keys have `esy: {}`. We could change them to `pjc: {}` before flipping them over to the master branches, if that saves us one round of thrash, but maybe we can even convert them to their final form right now, if you have a good idea what this field should be called.

Some desirable characteristics of a `package.json` field name:

- Immediately clear that the field is intended to integrate with some other, specific and semi-formalized system. (As much as I love the field name `build: {}` it's not clear that the field conforms to some specification. `pjc` is clearly something specific, but not searchable.
-  `yarn:{}` might be a good candidate, but some might get the incorrect idea that we're not open to other package build systems (such as our own internal deployment build servers) also adopting the same spec. (It might still be worth creating it under `yarn: {}` though).

Importantly, I'd like to minimize thrash so if we can get it right from day one, all the better!